BambooTracker/res/lang/bamboo_tracker_fr.ts

```diff
@@ -551,32 +551,32 @@
     <message>
         <location filename="../../gui/fm_envelope_set_edit_dialog.ui" line="14"/>
         <source>Edit FM envelope set</source>
-        <translation type="unfinished"></translation>
+        <translation>Éditer le format d&apos;enveloppe FM</translation>
     </message>
     <message>
         <location filename="../../gui/fm_envelope_set_edit_dialog.ui" line="39"/>
         <source>Add</source>
-        <translation type="unfinished">Ajouter</translation>
+        <translation>Ajouter</translation>
     </message>
     <message>
         <location filename="../../gui/fm_envelope_set_edit_dialog.ui" line="52"/>
         <source>Set digit types in the order of appearence:</source>
-        <translation type="unfinished"></translation>
+        <translation>Typer les valeurs d&apos;après l&apos;ordre d&apos;apparition :</translation>
```

Comment:
bizarre ce "typer", j'aurai mis un truc du genre "paramétrer les types numériques d'après l'ordre d'apparition"

Reply:
Not really, original text would mean "types de chiffres" rather (hex digits); "types numériques" is not appropriate for the context.

But also in original english text, I believe "digit" is not really appropriate either; because the values are really "numbers" not digits.
It's a reason why I chose to translate as "valeur" ("value").

"Typer" means to associate something a type, which is the purpose of the editing.
I don't mind changing it to something else. example "Définir les types de valeurs d'après l'ordre d'apparition"

(Also by default I try to favor the shorter texts in prevention of any overflows. In this case any will be fine, I just verified)

Reply:
> Définir les types de valeurs d'après l'ordre d'apparition

seems much better from a common user point of view